CHANGELOG.md

```diff
@@ -1,35 +1,35 @@
-# [v5.6.3-nightly.12](https://github.com/getferdi/ferdi/compare/v5.6.3-nightly.10...v5.6.3-nightly.12) (2021-09-24)
+# [v5.6.3-nightly.12](https://github.com/ferdium/ferdium-app/compare/v5.6.3-nightly.10...v5.6.3-nightly.12) (2021-09-24)
```

Comment:
All the PR links in the README are broken (404 ATM, but will link to the wrong PR when it will be created). I think we should remove them.